src/main/java/com/ibm/watson/modelmesh/Metrics.java

```diff
@@ -230,6 +230,17 @@ public PrometheusMetrics(Map<String, String> params) throws Exception {
                 }
             }
 
+            // Custom info metrics
+            if (!customParams.isEmpty()){
+                Gauge customMetrics = Gauge.build()
+                        .name(customParams.get("metric_name"))
+                        .help("Info Metrics")
+                        .labelNames("deployment", "slot", "component", "group")
+                        .create();
+                customMetrics.labels(customParams.get("deployment"), customParams.get("slot"), customParams.get("component"), customParams.get("group"));
+                registry.register(customMetrics);
+            }
```

Comment:
@rafvasq we don't want to hardcode label names here. The whole idea is that the label names/values are provided in the env var that you're parsing. We can enforce a maximum number of them (e.g. 4 or 5).